Create countdown in console

Matt-1985 opened the pull request:
Add the countdown in JavaScript

Matt-1985 committed:
Create timer in console

lmachens commented on src/main.js:
```diff
@@ -0,0 +1,18 @@
+    //dadaurch sage ich wenn die zeit 0 ist wird gestoppt
+    clearInterval(timeOut);
+
+    return console.log("Ende");
```
`return` has no effect here

Matt-1985 replied:
I will change it than, thank you leon

lmachens commented on src/main.js:
```diff
@@ -0,0 +1,18 @@
+  }
+};
+
+let timeOut = window.setInterval(countdown, 1000); //damit wird garantiert das die funktion sich immer wiederholt
```
`window` is optional in this case because `window` is your global variable which is accessible from anywhere.
`setInterval` returns an intervalId, `setTimeout` returns a timeoutId.
I would rename the variable.

```suggestion
let intervalId = setInterval(countdown, 1000); //damit wird garantiert das die funktion sich immer wiederholt
```

lmachens commented on src/main.js:
```diff
@@ -0,0 +1,18 @@
+
+const countdown = () => {
+  console.log(time); //dadurch sehen wir die "20" als Anfangswert
+  time = time - 1; //dadurch sage ich das immer 1 abgezogen werden soll
```
Alternative:
```suggestion
  time--; //dadurch sage ich das immer 1 abgezogen werden soll
```

lmachens reviewed (commented):
Good job! 
I ignored the comments and focused on the JS.

Matt-1985 committed:
Update src/main.js

Thank you very much :)

Co-authored-by: Leon Machens <leon.machens@googlemail.com>

Matt-1985 committed:
Update src/main.js

Thank you for the tip! I'll change it ...and keep to think about the next time :)

Co-authored-by: Leon Machens <leon.machens@googlemail.com>

Matt-1985 committed:
Add css base

Matt-1985 committed:
Merge branch 'consoleTimer' of github.com:Matt-1985/countdown_timer into consoleTimer
